examples/20_basic/simple_datasets_tutorial.py

```diff
@@ -38,13 +38,13 @@
 # Load a dataset
 # ==============
 
-# X - An array/dataframe where each row represents one example with
+# features - An array/dataframe where each row represents one example with
 # the corresponding feature values.
-# y - the classes for each example
+# labels - the classes for each example
 # categorical_indicator - an array that indicates which feature is categorical
-# attribute_names - the names of the features for the examples (X) and
-# target feature (y)
-X, y, categorical_indicator, attribute_names = dataset.get_data(
+# attribute_names - the names of the features for the examples (features) and
```

Comment:
It would be good to replace the features in `the names of the features` with attributes?